Dockerfile

```diff
@@ -209,6 +226,11 @@ RUN --mount=type=cache,target=/.cache go generate ./...
 RUN goimports -w -local github.com/talos-systems/talos ./
 RUN gofumpt -w ./
 
+FROM build AS fix_embed
```

Comment:
nit: `fix-embed` (we don't use underscores)

Reply:
Missed that. Going to fix in the desk Dockerfile update.